.github/workflows/ci.yml

```diff
@@ -9,8 +9,6 @@ jobs:
         os: [ubuntu-latest]
         node-version:
           - 14.x
-          - 16.x
```

Comment:
Change not needed?

Reply:
Hey Ruben, that was just for testing purposes since I wasn't able to reproduce the issues locally (and didn't want the succeeding tests to be re-run to save some resources).

Reply:
Ah ok. Just ping me when this is ready for review :)